integration/test/basic/project.yaml

```diff
@@ -0,0 +1,5 @@
+version: 1
+
+test:
+  env:
+    - "VAULT_TOKEN=myToken"
```

Comment:
Is it used?

Reply:
Indeed, I've proposed https://github.com/giantswarm/e2e-harness/pull/92 to make things more clear.

Reply:
Sorry I got confused. So it can be removed?

Reply:
Proposed here https://github.com/giantswarm/certctl/pull/70.

Reply:
Ack. I'd prefer if the project.yaml changes were introduced with #70. That would be easier to follow I think.
Thanks for explanations.